core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -1026,12 +1027,18 @@ public boolean isFormatted() {
   public void format() throws IOException {
     File journalPath = mConf.getPath();
     if (journalPath.isDirectory()) {
-      org.apache.commons.io.FileUtils.cleanDirectory(mConf.getPath());
+      if (alluxio.util.io.FileUtils.isStorageDirAccessible(journalPath.getPath())) {
```

Comment:
is this due to a new finding?

Reply:
Yes, you inspire me. **`FileUtils.cleanDirectory()` will not delete the dir but just clean its children.** If journalPath is a no privileged directory and exists. but journalPath contains none file or dir. The job server service will hang on. I test it on this condition.